### ISSUE TITLE
Override weapon sway/bobbing for mods which do weird high FPS fixes

### ISSUE THREAD

**Issue by Skorpjun161** [shinyquagsire23/OpenJKDF2#311]
The weapon sway / bobbing while moving is missing. You can see how the weapon sways in the original at the URL below:
https://youtu.be/9pvk_BWKXtU?t=720
In OpenJKDF2 it doesn't sway at all:
https://youtu.be/rMqioXVxAd0?t=1689

**Comment by shinyquagsire23** [comment]
That might be the fault of a high-res mod which tried to fix the bobbing being too fast -- as part of my high framerate fixes, I fixed the bobbing w/o needing to use a mod.

**Comment by GeneralTantor** [comment]
@shinyquagsire23 Yeah if you can let us know which mod is causing it we can remove it from the next release. Thanks Max!

**Comment by shinyquagsire23** [comment]
Actually, I apparently can recreate this, it seems to be jknupgfx.gob. They must have bundled it into their mod for whatever reason.

I could probably try and clamp the sway or detect the fixup mods, but then there's the risk that some mod *intentionally* sets values that makes it not sway.

See also: https://github.com/jdmclark/jkgfxmod/issues/71#issue-927518664

**Comment by GeneralTantor** [comment]
Ok thanks ill see if I can edit that Gob file and remove whatever is stopping it from swaying

**Comment by GeneralTantor** [comment]
Strange I set the code in the weap_bryar_m.cog file to `jkSetWaggle(player, '10.0 7.0 0.0', 350);` but it didnt change anything not sure what I did wrong... Testing in MOTS version.

**Comment by GeneralTantor** [comment]
I will reach out to the creator of that mod see what is up

**Comment by nikumubeki** [comment]
The weapon COGs in my EMJK/EMSITH compilation mods (https://www.moddb.com/mods/todoa/downloads) were made for JkGfxMod, so their jksetwaggle values have been minimized quite a bit to accommodate for how insanely fast the weapon waggle was with that mod. With openjkdf2, if you revert all the jksetwaggle values to what they were in the original COGs from JKMRES.goo or Res2.gob, they should be as fast as they were originally meant to be.

As a test, if you have EMSITH installed on an OpenJKDF2 MotS directory and extract this zip file to the root directory, the bryar and blastech pistols should now waggle properly, so the rest of the weapon COGs can be edited in the same way (just make sure that you match all the waggle values with the original and that there are no conflicting cog files of the same name being loaded).

[Resource.zip](https://github.com/shinyquagsire23/OpenJKDF2/files/14816027/Resource.zip)

**Comment by GeneralTantor** [comment]
Awesome Man! Fastgamerr does it again! You Rock Man!

**Comment by GeneralTantor** [comment]
> The weapon sway / bobbing while moving is missing. You can see how the weapon sways in the original at the URL below: https://youtu.be/9pvk_BWKXtU?t=720 In OpenJKDF2 it doesn't sway at all: https://youtu.be/rMqioXVxAd0?t=1689

Hey man Using Fastgamerr's example above I was able to fix this! It will be in the next release of remastered mod
